.github/workflows/veracode-analysis.yml

```diff
@@ -73,10 +69,15 @@ jobs:
 
           sudo ninja
 
+          echo "[DEBUG] - Find compiled files"
+          find ./ -name "*.so"
+          echo "[DEBUG] - Build size"
+          du -sh ./lib/*.so | sort -rh
+
       - name: Binary preparation
         run: |
           echo "[INFO] - Keeping only compiled files"
-          find ./build -type f -not \( -name "*.so" -o -name "*.h" -o -name "*.cc" -o -name "*.hpp" \) -delete
+          find ./build -type f -not \( -name "*.so" \) -delete
```

Comment:
Maybe I'm going to say something totally stupid. You're warned !
Here you remove in build files that are not *.so
But we also have binaries in build/bin...
Don't you check them?
And in build/lib, we have *.a files that are static libraries, so equivalent to *.so but static instead of dynamic.

I have the feeling you don't check everything...

Reply:
Indeed, the first objective of this task was to parallelize the analysis.
In veracode I could only find .so file analysed. But we can try to add others.
We can also trick the engine to force the analysis by renaming the static libraries from .a to .so 🤔

Reply:
to add : 
/build/bin (at least cbd)

Reply:
added to binary to analyze